shell/renderer/api/atom_api_web_frame.cc

```diff
@@ -110,32 +111,57 @@ class RenderFrameStatus final : public content::RenderFrameObserver {
 
 class ScriptExecutionCallback : public blink::WebScriptExecutionCallback {
  public:
+  // for compatibility with the older version of this, error is after result
```

Comment:
I think this is worth mentioning in the docs, too, as it's contrary to node style.